eng/pipelines/templates/stages/archetype-net-release.yml

```diff
@@ -0,0 +1,221 @@
+parameters:
+  Artifacts: []
+  ArtifactName: 'not-specified'
+
+stages:
+  - stage: Signing
+    dependsOn: ${{parameters.DependsOn}}
+    jobs:
+      - template: pipelines/jobs/sign-package.yml@azure-sdk-build-tools
+        parameters:
+          ArtifactName: ${{parameters.ArtifactName}}
```

Comment:
I think it would be better if we have the signing project simply take a directory which contains the nupkg files that need to be signed. It shouldn't need to know about how to extract the artifacts. Eventually we plan to change that somewhat and I'd rather do that all in one place which is here and have the signing template only care about the signing logic.